packages/cli/src/net/clientContext.mts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface ClientContext {
```

Comment:
I'm worried this is going to cause confusion with the existing shared `ClientContext` and `createClientContext` in `shared.net` especially as we import and use them already within this package. Can we name these something slightly different to make it clearer these are just used for the internal handwritten clients of this package? For example `InternalClientContext` or something